File: 1580-ShuffleTheArray/1580-ShuffleTheArray.py
# Last updated: 7/30/2025, 12:23:13 PM
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        ans = []
        for i in range(0,n):
            ans.extend([nums[i],nums[n+i]])
        return ans

File: 1580-ShuffleTheArray/1580-ShuffleTheArray.java
// Last updated: 7/30/2025, 12:23:08 PM
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] ans = new int[nums.length];
        for(int i=0; i<n; i++){
            ans[2*i] = nums[i];
            ans[2*i+1] = nums[n+i];
        }
        return ans;
    }
}